compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/reference/AnvilModuleDescriptor.kt

```diff
@@ -23,47 +22,50 @@ public interface AnvilModuleDescriptor : ModuleDescriptor {
     lookupLocation: LookupLocation = FROM_BACKEND
   ): ClassDescriptor?
 
-  public fun getClassesAndInnerClasses(ktFile: KtFile): List<KtClassOrObject>
-
-  public fun getKtClassOrObjectOrNull(fqName: FqName): KtClassOrObject?
```

Comment:
My goal was to get rid of these APIs. We don't want to work with `KtClassOrObject` anymore.